config/puma.rb

```diff
@@ -1,3 +1,26 @@
 # frozen_string_literal: true
 
 plugin :statsd
+
+# Configure "min" to be the minimum number of threads to use to answer
+# requests and "max" the maximum.
+#
+# The default is "0, 16".
+#
+threads 32, 32
+
+# === Cluster mode ===
+
+# How many worker processes to run.
+#
+# The default is "0".
+#
+workers 2
```

Comment:
Workers ~= memory. Normally you'd choose 3 (unless you're on heroku which is 1-2 since memory is at a premium) assuming we have the memory. Rails is ~300mb most of the time, so 1gb tends to be the default sizing when you get to choose.

@jbritt1 what size EC2 instance are we running here?

Reply:
This was a t3.medium (what was in staging at the time). I have since updated staging to reflect what sits in prod (t3.large) [here](https://github.com/department-of-veterans-affairs/devops/pull/6164/files#diff-ecbe20a47ab05abd40f0c4373647a772L20).